ui/lib/core/addon/components/form-field.hbs

```diff
@@ -157,6 +157,41 @@
             {{/if}}
           </Hds::Form::TextInput::Field>
         {{/if}}
+        {{#if (eq @attr.options.editType "textarea")}}
```

Comment:
[consideration] to make it easier to follow the code nesting, where it starts and where it ends, I would use an `{{else if}}` here (also, this would make it more similar to the original code when we started the migration)